src/resolve/index.ts

```diff
@@ -6,17 +6,73 @@ import {Got} from '../network/got'
 import {Package} from '../types'
 import {LoggedPkg} from 'pnpm-logger'
 
-export type ResolveResult = {
```

Comment:
I like `Resolution` more that `ResolveResult` but I can revert the naming, thoughts?